__tests__/app.test.js

```diff
@@ -358,31 +360,66 @@ describe("GET /api/articles?topics=", () => {
       .then(({ body }) => {
         expect(body.length).toBe(1);
         body.forEach((article) => {
-          expect(typeof article.title).toBe('string'),
-          expect(article.topic).toBe('cats'),
-          expect(typeof article.author).toBe('string'),
-          expect(typeof article.body).toBe('string'),
-          expect(typeof article.article_img_url).toBe('string')
-        }
-      );
-  });
-})
-test("404: invalid endpoint", () => {
-  return request(app)
-    .get(`/api/articlesssss?topic=cats`)
-    .expect(404)
-    .then(({ body }) => {
-      const { message } = body;
-      expect(message).toBe("path not found!");
-    });
-});
-test("404: responds with error message when passed a topic that doesn't have any articles related to it", () => {
-  return request(app)
-    .get(`/api/articles?topic=catssss`)
-    .expect(404)
-    .then(({ body }) => {
-      const { message } = body;
-      expect(message).toBe("path not found!");
-    });
+          expect(typeof article.title).toBe("string"),
+            expect(article.topic).toBe("cats"),
+            expect(typeof article.author).toBe("string"),
+            expect(typeof article.body).toBe("string"),
+            expect(typeof article.article_img_url).toBe("string");
+        });
+      });
+  });
+  test("404: invalid endpoint", () => {
+    return request(app)
+      .get(`/api/articlesssss?topic=cats`)
+      .expect(404)
+      .then(({ body }) => {
+        const { message } = body;
+        expect(message).toBe("path not found!");
+      });
+  });
+  test("404: responds with error message when passed a topic that doesn't have any articles related to it", () => {
+    return request(app)
+      .get(`/api/articles?topic=catssss`)
+      .expect(404)
+      .then(({ body }) => {
+        const { message } = body;
+        expect(message).toBe("path not found!");
+      });
+  });
 });
-});
\ No newline at end of file
+
+describe("GET /api/articles/:article_id + comment_count", () => {
+  test("200: responds with an article with the related article_id and adds a comment count", () => {
+    return request(app)
+      .get(`/api/articles/1`)
+      .expect(200)
+      .then(({ body }) => {
+          expect(typeof body.author).toBe("string");
+          expect(typeof body.title).toBe("string");
+          expect(typeof body.article_id).toBe("number");
+          expect(typeof body.topic).toBe("string");
+          expect(typeof body.created_at).toBe("string");
+          expect(typeof body.votes).toBe("number");
+          expect(typeof body.article_img_url).toBe("string");
+          expect(typeof body.comment_count).toBe("string");
+        });
+      });
```

Comment:
The following 2 tests are a repeat of the same tests when you did get /api/articles/:article_id